backend-tests/tests/test_artifact.py

```diff
@@ -565,6 +576,10 @@ def test_artifacts_exclusive_to_user(self, mongo, clean_mongo):
             api_client.with_auth(auth_token)
             r = api_client.call("GET", deployments.URL_DEPLOYMENTS_ARTIFACTS)
             assert r.status_code == 200
-            artifacts = r.json()
+            artifacts = [
+                artifact
+                for artifact in r.json()
+                if not artifact["name"].startswith("mender-demo-artifact")
```

Comment:
no, this is a problem today as well, because in the staging environment (which is configured as the production one) when you create a tenant, you automatically get the demo artifact. The test failed, and I had to exclude the demo artifact when counting.